lib/utils/test-task-mixin.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       testemOptions.file = 'testem-electron.js';
     }
     else {
-      deprecate("In ember-cli v3.2.0 and newer, testemOptions is now called defaultOptions. Please consider upgrading to use the new method.", false);
+      deprecate("In ember-cli v3.2.0 and newer, testemOptions is now called defaultOptions. Please consider upgrading to use the new method.", true);
```

Comment:
I don't think you need this deprecation warning here.

If the host application is using `ember-cli < v3.2`, then `this._super(...arguments)` will return a `testemOptions` object and everything will work fine (and the else clause won't be executed). If the host application is using `ember-cli >= 3.2` then this method won't be called at all, the else clause (and the rest of the function) won't be executed.

You're upgrading code inside the addon to be compatible with both old and new `ember-cli`, so there's nothing to deprecate...

Reply:
I guess I thought consumers might be calling `testemOptions`, in which case they'd want to migrate to the new API at some point. I'm probably over-thinking this though. If it's just internal then you're right, we should probably just use if-else/try-both logic in there and go on our merry way.